ruby/cucumber-gherkin.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
     'source_code_uri' => 'https://github.com/cucumber/gherkin/blob/main/ruby'
   }
 
-  s.add_runtime_dependency 'cucumber-messages', '>= 19.1.4', '< 23'
+  s.add_runtime_dependency 'cucumber-messages', '>= 19.1.4', '< 24'
```

Comment:
I am curious: does this marker mean "use the gem less than 24", while the title of the PR is "update to v24"? What's the right value to use?